visual/icons/CodeOptions.svg

```diff
@@ -0,0 +1 @@
+<svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><path transform="translate(7.5-3) rotate(45)" d="m7 1c-.554 0-1 .446-1 1v2h4v-2c0-.554-.446-1-1-1zm-1 4v7l2 3 2-3v-7zm1 1h1v5h-1z" fill="#def"/></svg>
```

Comment:
```suggestion
<svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><path transform="rotate(45 8 8)" d="M7 1a1 1 0 0 0 -1 1v2h4v-2a1 1 0 0 0 -1-1zm-1 4v8l2 3 2-3v-8zm1 1h1v6h-1z" fill="#def"/><path d="M9 6l-5 5" fill="none" stroke="#def" stroke-opacity=".4" stroke-width="2"/></svg>
```

Optimized and a bit tweaked to not be a carbon copy of Godot's Edit.svg

Reply:
I agree this does look way better, thanks!